src/tests/ftest/container/boundary.py

```diff
@@ -0,0 +1,143 @@
+    :avocado: recursive
+    """
+
+    def create_container_and_test(self, pool=None, pool_num=1, cont_num=1, with_io=False):
+        """To create single container on pool.
+        Args:
+            pool (str): pool handle to create container.
+            pool_num (int): pool number.
+            container_num (int): container number to create.
+            with_io (bool): enable container test with execute_io.
+        """
+
+        try:
+            container = self.get_container(pool)
+        except (DaosTestError, TestFail) as err:
+            self.fail("#(3.{0}.{1}) container create failed. err={2}".format(
+                pool_num, cont_num, err))
+
+        self.log.info("===(3.{0}.{1})create_container_and_test, container {2} created..".format(
+            pool_num, cont_num, container))
+        if with_io:
+            io_run_time = self.params.get("run_time", '/run/container/execute_io/*')
+            rank = self.params.get("rank", '/run/container/execute_io/*')
+            obj_classs = self.params.get("obj_classs", '/run/container/execute_io/*')
+            try:
+                data_bytes = container.execute_io(io_run_time, rank, obj_classs)
+                self.log.info(
+                    "===(3.{0}.{1})Wrote {2} bytes to container {3}".format(pool_num, cont_num,
+                    data_bytes, container))
+            except (DaosTestError, TestFail) as err:
+                self.fail("#(3.{0}.{1}) container IO failed, err: {2}".format(
+                    pool_num, cont_num, err))
+        time.sleep(2)  #to syncup containers before close
+
+        try:
+            self.log.info("===(4.{0}.{1})create_container_and_test, container closing.".format(
+                pool_num, cont_num))
+            container.close()
+            self.log.info("===(4.{0}.{1})create_container_and_test, container closed.".format(
+                pool_num, cont_num))
+        except (DaosTestError, TestFail) as err:
+            self.fail("#(4.{0}.{1}) container close fail, err: {2}".format(
+                pool_num, cont_num, err))
+
+    def create_containers(self, pool=None, pool_num=10, num_containers=100, with_io=False):
+        """To create number of containers parallelly on pool.
+        Args:
+            pool(str): pool handle.
+            pool_num (int): pool number to create containers.
+            num_containers (int): number of containers to create.
+            with_io (bool): enable container test with execute_io.
+        """
+
+        self.log.info("==(2.{0})create_containers start.".format(pool_num))
+        thread_manager = ThreadManager(self.create_container_and_test, self.timeout - 30)
+
+        for cont_num in range(num_containers):
+            thread_manager.add(
+                pool=pool, pool_num=pool_num, cont_num=cont_num, with_io=with_io)
+        self.log.info("==(2.{0}) {1} containers created.".format(pool_num, num_containers))
```

Comment:
(pylint-logging-format-interpolation) Use lazy % formatting in logging functions